t/plugin/dynamic-upstream.t

```diff
@@ -25,14 +25,15 @@ run_tests;
 
 __DATA__
 
-=== TEST 1: route binding routex plugin
+=== TEST 1: route binding dynamic-upstream plugin
 --- config
     location /t {
         content_by_lua_block {
             local t = require("lib.test_admin").test
             local code, body = t('/apisix/admin/routes/1',
                 ngx.HTTP_PUT,
                 [[{
+                    "uri": "/hello",
```

Comment:
use `/server_port`, need to confirm the upstream node

Reply:
fixed.